app/assets/javascripts/mumuki_laboratory/application/gamification.js

```diff
@@ -47,3 +47,9 @@ class LevelProgression {
     return (this.currentExp - baseExpCurrentLevel) / (baseExpNextLevel - baseExpCurrentLevel);
   }
 }
+
+(function (mumuki) {
```

Comment:
Should I change this to `(() => {`? @flbulgarelli :smile:

Reply:
That would be nice :smile: